packages/ra-data-fakerest/src/index.ts

```diff
@@ -134,7 +134,8 @@ export default (data, loggingEnabled = false): DataProvider => {
             return Promise.reject(error);
         }
         if (loggingEnabled) {
-            log(type, resource, params, response);
+            const { signal, ...paramsWithoutSignal } = params;
```

Comment:
I also fixed the log for fakerest - it used to show the `signal` parameter in every query, which is useless in this case.